runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeItemPanel.java

```diff
@@ -53,7 +53,7 @@
 	private static final Dimension ICON_SIZE = new Dimension(32, 32);
 
 	GrandExchangeItemPanel(AsyncBufferedImage icon, String name, int itemID, int gePrice, Double
```

Comment:
Why are you not using the primitive "double"?

Reply:
I'm not sure why it was originally coded that way, but my guess is to take advantage of the built in intValue() function, which is called in line 156, and I reused in line 164. Going to the implementation, I see that the function just typecasts the variable as int, but I think Double.intValue() is a cleaner look than int(double)